src/datasources/errors/http-error-factory.spec.ts

```diff
@@ -24,7 +24,9 @@ describe('HttpErrorFactory', () => {
 
     expect(actual instanceof DataSourceError).toBe(true);
```

Comment:
Here we know that `actual` is a `DataSourceError` so we can avoid the `instanceof` operator

Reply:
I removed all `instanceof `checks in 646f900c7186f2e596033fe23ebb9e1a8202c7bf.